changelog.txt

```diff
@@ -1,3 +1,20 @@
+4.44.0
+==========
+### New Features
+**Added support in MSAL for dSTS authority** See [3198](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3198).
+**Enabled Azure.Identity to benefit from MSAL.NET token cache when used for Managed identity** See [3137](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3137).
+**Added PPE to known discovery endpoints** See [3265](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3265).
+**Refactored the performance tests and create YAML pipeline to run with CI Build** See [3297](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/pull/3297).
+
+### Experimental Features
+**MSAL.NET now have WAM integration via MSAL Runtime with AT POP support** See [3192](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3192).
```

Comment:
```suggestion
**MSAL.NET now has new WAM integration via MSAL Runtime with support for Proof-of-Possession access tokens** See [3192](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3192) and [wiki](http://aka.ms/msal-net-wam).
```

@gladjohn What's an official public name for MSAL runtime? (It went through a few iterations... Would be good to make sure we're consistent in naming it.)

Reply:
[Per our docs](https://aka.ms/msal-net-wam) "The New MSAL WAM Preview is an abstraction layer based on MSAL C++ "